### PR TITLE
Add recompiled cython files

### PR DESCRIPTION
**Pull request by ArvydasVe**

### CODE REVIEW

File: ssh2/utils.c [hunk 1]
```diff
@@ -7,8 +7,8 @@
 #elif PY_VERSION_HEX < 0x02060000 || (0x03000000 <= PY_VERSION_HEX && PY_VERSION_HEX < 0x03030000)
     #error Cython requires Python 2.6+ or Python 3.3+.
 #else
-#define CYTHON_ABI "0_29_6"
-#define CYTHON_HEX_VERSION 0x001D06F0
+#define CYTHON_ABI "0_29_7"
```
Comment (cj-matti-niemenmaa) [inline ssh2/utils.c]: Is there some possibility for problems due to this version being different?

Comment (ArvydasVe) [reply ssh2/utils.c]: Looking at the changes made to these files I don't see many problem possibility, but to be sure I should install the older cython and recompile.

Comment (ArvydasVe) [reply ssh2/utils.c]: Done

**Review by cj-matti-niemenmaa — APPROVED** [review]
This diff is too big to really review, I'll just assume that it's correct.

**Comment by Red-M** [comment]
Can the `method_pref` function be merged into the main repo for ssh2-python?
I've got a repo ready that could be merged into the main ssh2-python repo with this feature and also updating libssh2 to the latest stable version.

**Pull request merged**